guides/05_Simulcast.md

```diff
@@ -52,25 +50,6 @@ The table entries represent the maximum configuration.  When CPU and bandwidth c
 
 Note that simulcast is disabled when there are only 2 or fewer attendees.  This is because WebRTC has additional functionality to request lower bitrates from the remote end, and we will forward these requests if there are no competing receivers (i.e. if the receiving estimates it has 200kbps downlink bandwidth available, this will be sent and relayed in a message to the sending client).  Therefore there is no need for simulcast based adaption.
 
-### Downlink  Bandwidth Policy
-
-The [VideoAdaptiveProbePolicy](https://aws.github.io/amazon-chime-sdk-js/classes/videoadaptiveprobepolicy.html) adds functionality to take advantage of video simulcast.  The goal of this policy is to leave the complexity of managing the downlink bandwidth and decision of which streams to request inside the SDK and remove that burden from the application.  The policy monitors numerous pieces of information and uses that to determine which streams to subscribe to.  The data considered are:
-
-1. Message from infrastructure enumerating video streams from each remote client
-2. Estimated downlink bandwidth from WebRTC library
-3. Media metrics such as packet loss and used bandwidth
```

Comment:
This section has way too many lower level details that builders don't need, so i didn't move them over to the other guide.